src/main/menu.ts

```diff
@@ -311,6 +311,13 @@ function getMenuTemplate(logHandler: LogHandler): rawMenuItem[] {
           },
           accelerator: 'F1',
         },
+        {
+          translate: 'keybindings',
+          click: () => {
+            mainWindow.send('showKeybindingsDialog')
+          },
+          accelerator: isMac ? 'Cmd+/' : 'Ctrl+/',
```

Comment:
it does work, but it's not where the user assumes because apple has strange keybindings for german

Reply:
> it does work, but it's not where the user assumes

hm, do you think, many user will assume `Cmd+/` for the keybindings just so?

therefore, i think, it is better to add the hint for an actually working shortcut and not something that is known to not work on quite some systems (i doubt it is only german).

alternatively, if `Cmd+/` is that known and standard - just do not show a shortcut at all in the menu and in the keybindings. this would at least not result in other ppl thinking there is a bug.

Reply:
> just do not show the shortcut at all in the menu and in the keybindings. this would at least not result in other ppl thinking there is a bug.

then its a bug the shortcut is not there...

> maybe it is better to add the hint for an actually working shortcut and not something that is known to not work on quite some systems (i doubt it is only german).

Discord has this "issue" too and they don't seem to care.

Reply:
> then its a bug the shortcut is not there...

but this is quite different, or? to show a shortcut that is not working is not the same kind of issue as just having an additional shortcut that is not printed explicitly.

but well, i agree that there are lower hanging fruits to pick, i just try to suggest things that look less buggy without adding much more work  :)